MAINTAINERS_RULES.md

```diff
@@ -0,0 +1,39 @@
+# Maintainers Rules
+
+This document lays out some basic rules and guidelines all maintainers are expected to follow.
+Changes to the [Acceptance Criteria](#hard-acceptance-criteria-for-merging-a-pr) for merging PRs require a ceiling(two-thirds) supermajority from the maintainers.
+Changes to the [Repo Guidelines](#repo-guidelines) require a simple majority.
+
+## Hard Acceptance Criteria for merging a PR:
+
+- 2 LGTMs are required when merging a PR
```

Comment:
2 LGTMs: does this mean 2 approvals from the maintainers group, 2 approvals from the contributors group, or a combination?

Reply:
I think at least 1 LGTM from someone in the maintainer group is desired. Of course all community review is welcome.

Reply:
I would like to see that more explicitly specified here.

Reply:
OK, I think 1 LGTM from a maintainer should be the bar.

Reply:
i would still like 2 LGTM from maintainers. One should LGTM/approve and the other one should approve and merge. If we dont have enough maintainers, we should promote more. Specially someone outside of Quantum and who wasnt the CTO of Quantum :)